bundles/org.openhab.core/src/test/java/org/openhab/core/types/util/UnitUtilsTest.java

```diff
@@ -120,6 +120,7 @@ public void testGetDimensionNameWithDimension() {
 
         unit = Objects.requireNonNull(UnitUtils.parseUnit("m"));
         assertThat(UnitUtils.getDimensionName(unit), is(Length.class.getSimpleName()));
+        assertThat(UnitUtils.getDimensionName(Units.MIRED), is(Temperature.class.getSimpleName()));
```

Comment:
Perhaps add the following line to assert that nothing got broken due to the new inversion test..

```
assertThat(UnitUtils.getDimensionName(Units.KELVIN), is(Temperature.class.getSimpleName()));
```